### PR TITLE
Pylint and continuous integration

### PR DESCRIPTION
**Pull request by Saityi**
@sourlows test running thingy and pylint running thingy

### CODE REVIEW

File: src/pylintrc [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+[TYPECHECK]
+ignored-classes=google.appengine.api.memcache
+[MISCELLANEOUS]
+notes=FIXME,XXX,TODO
+[REPORTS]
+# no reports makes failures easier to track down
+reports=n
+include-ids=y
+[MASTER]
+# Can safely ignore libs - VBC doesn't manage the code that lives in these dirs.
```
Comment (sourlows) [inline src/pylintrc]: delete this comment

File: test/pylintrc [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+[TYPECHECK]
+ignored-classes=google.appengine.api.memcache
+[MISCELLANEOUS]
+notes=FIXME,XXX,TODO
+[REPORTS]
+# no reports makes failures easier to track down
+reports=n
+include-ids=y
+[MASTER]
+# Can safely ignore libs - VBC doesn't manage the code that lives in these dirs.
```
Comment (sourlows) [inline test/pylintrc]: delete this comment

**Comment by sourlows** [comment]
can you explain how to run these in README.md ? Delete those comments and merge this in when you're ready. Good job!

File: test/api/league_test.py [hunk 3]
```diff
@@ -89,22 +88,28 @@ def test_post_returns_400_if_invalid_rating_scheme_is_specified(self):
         self.assertEqual(response.data, '"Invalid rating scheme %s"\n' % self.args_mock.return_value['rating_scheme'])
 
     def test_post_new_league_when_created_successfully(self):
-        self.create_test_league()
-        self.mock_function_in_setup('app.api.league.create_league', return_value=self.league)
+        self.create_test_user()
```
Comment (Saityi) [inline test/api/league_test.py]: @sourlows Mocking create_league() for this test bugged out when I changed stuff in src/main.py (??). I'm iffy on the way I changed it, but I couldn't find a way to make it correctly test post as written before.

**Pull request merged**